app/mutations/FormChange/updateFormChange.ts

```diff
@@ -13,7 +13,6 @@ const mutation = graphql`
         changeStatus
         projectRevisionByProjectRevisionId {
           ...TaskList_projectRevision
-          totalProjectValue
```

Comment:
I don't ~*think I need to add the calculated values to this file because `updateFormChange` isn't called in the funding form anywhere